### PR TITLE
OPS-0 Add TF version 1.0

### PR DESCRIPTION
**Pull request by snovikov**

### CODE REVIEW

**Review by ronny-panknin-flaconi — APPROVED** [review]

File: .github/workflows/docker_build.yml [hunk 1]
```diff
@@ -22,18 +22,19 @@ jobs:
...
       matrix:
         TF_VERSION:
-          - 0.12
-          - 0.13
-          - 0.14
-          - latest
+          - "0.12"
+          - "0.13"
+          - "0.14"
+          - "1.0"
```
Comment (cytopia) [inline .github/workflows/docker_build.yml:28]: Currently only Terragrunt version 0.31 supports Terraform 1.0, which I am not seeing in this list

https://github.com/gruntwork-io/terragrunt/releases/tag/v0.31.0

**Review by cytopia — COMMENTED** [review]
I would also suggest building the matrix as such, because currently we do have TF versions that do not work with TG versions.

See working approach here: https://github.com/cytopia/docker-terragrunt/blob/master/.github/workflows/build.yml#L32

File: .github/workflows/docker_build.yml (hunk 1 above)
Comment (ronny-panknin-flaconi) [reply .github/workflows/docker_build.yml:28]: the change was only from double/float to string, no new version was added here ;)

**Comment by cytopia** [comment]
@snovikov closing in favour of recent rewrite

**Pull request closed without merging**